### PR TITLE
Added missing Process fields; More accurate Optional parameters

### PR DESCRIPTION
**Pull request by eujing**
Just started out using mypy on a project using multiprocessing, and noticed some Processing fields that should be available turn up as errors, e.g. "Process" has no attribute "exitcode"

Looked up the docs from https://docs.python.org/3.6/library/multiprocessing.html#multiprocessing.Process and updated the .pyi file to more accurately represent it

### CODE REVIEW

File: stdlib/3/multiprocessing/__init__.pyi [hunk 1]
```diff
@@ -72,19 +72,25 @@ class Pool():
     def __exit__(self, exc_type, exc_val, exc_tb) -> None: ...
 
 class Process():
+    name: str = ...
```
Comment (JelleZijlstra) [inline stdlib/3/multiprocessing/__init__.pyi]: You don't need the ` = ...` here.

File: stdlib/3/multiprocessing/__init__.pyi [hunk 2]
```diff
@@ -72,19 +72,25 @@ class Pool():
...
+    daemon: bool = ...
+    pid: Optional[int] = ...
+    exitcode: Optional[int] = ...
+    authkey: bytes = ...
+    sentinel: int = ...
     # TODO: set type of group to None
     def __init__(self,
-                 group: Any = ...,
+                 group: Any = None,
```
Comment (JelleZijlstra) [inline stdlib/3/multiprocessing/__init__.pyi]: We actually prefer using `...` for all default values. In stubs, it doesn't matter exactly what the default is.

File: stdlib/3/multiprocessing/__init__.pyi [hunk 3]
```diff
@@ -72,19 +72,25 @@ class Pool():
...
                  target: Callable = ...,
-                 name: str = ...,
+                 name: Optional[str] = None,
                  args: Iterable[Any] = ...,
                  kwargs: Mapping[Any, Any] = ...,
-                 daemon: bool = ...) -> None: ...
+                 daemon: Optional[bool] = None) -> None: ...
```
Comment (JelleZijlstra) [inline stdlib/3/multiprocessing/__init__.pyi]: While you're at it, this argument is keyword-only (so there should be a `*` before daemon).

File: stdlib/3/multiprocessing/__init__.pyi [hunk 4]
```diff
@@ -72,19 +72,25 @@ class Pool():
...
+    daemon: bool = ...
+    pid: Optional[int] = ...
+    exitcode: Optional[int] = ...
+    authkey: bytes = ...
+    sentinel: int = ...
     # TODO: set type of group to None
     def __init__(self,
-                 group: Any = ...,
+                 group: Any = None,
                  target: Callable = ...,
```
Comment (JelleZijlstra) [inline stdlib/3/multiprocessing/__init__.pyi]: This is also Optional.

**Comment by eujing** [comment]
@JelleZijlstra ok, have updated based on your review!

**Comment by JelleZijlstra** [comment]
Thanks!

**Pull request merged**